src/pages/companion/companies/index.tsx

```diff
@@ -0,0 +1,57 @@
+import CompaniesList from "@/components/companion/CompaniesList";
+
+interface Company {
+    id: number;
+    name: string;
+    logo: string;
+    description: string;
+    tags: string[];
+    profile_url: string;
+}
+
+const companiesExample: Company[] = [
+    {
+        id: 1,
+        name: "Tesla Motors",
+        logo: "https://upload.wikimedia.org/wikipedia/commons/e/e8/Tesla_logo.png",
+        description: "3 delegates in attendance",
+        tags: ["AUTOMOTIVE", "R&D", "MECH ENGINEERING"],
+        profile_url: "/tesla"
```

Comment:
I noticed `CompanionItemRow` already has the template string `/companion/company/${company.profile_url}`, so we would end up with a double slash here: `/companion/company//tesla`